lib/rules/no-empty-component-block.js

```diff
@@ -85,6 +99,29 @@ module.exports = {
             isValueOnlyWhiteSpacesOrLineBreaks(componentBlock) ||
             componentBlock.children.length === 0
           ) {
+            emptyBlocks.push(componentBlock)
+          }
+        }
+
+        if (emptyBlocks.length === 0) return
+
+        if (autofix) {
+          const firstEmptyBlock = emptyBlocks[0]
+          context.report({
```

Comment:
I don't think the option changes the reported location.
They may be ignored by the `eslint-disable` directive, so I think it's necessary to report them one by one and auto fix them for each report.
Could you please change it?

Reply:
> Chose this approach is because I couldn't find a better way to handle reporting & fixing each emptyBlock within a for loop, as removing the first block will affect the range of the second block😵‍💫.

I think a good way to test removing them in one go is to have some whitespace between each block.
Adjacent autofixes will not be fixed at the same time to avoid conflicts.

Reply:
I'll look into it later, thanks.